spezi-invitation-code/firebase.json

```diff
@@ -0,0 +1,33 @@
+{
+  "firestore": {
+    "rules": "firestore.rules"
+  },
+  "storage": {
+    "rules": "firebasestorage.rules"
+  },
```

Comment:
We can remove the storage configuration here, I don't think we will be using it?